Mock.php

```diff
@@ -52,11 +52,11 @@ public function __construct(HttpClient $client, $latLong, $address)
         $this->address = $address;
 
         if (empty($this->latLong)) {
-            throw new \InvalidArgument('The Lat/Long must be set to use the Mock provider.');
+            throw new \InvalidArgumentException('The Lat/Long must be set to use the Mock provider.');
```

Comment:
I think the real typo here is actually in not using the `InvalidArgument` from Geocoder itself: `Geocoder\Exception\InvalidArgument`. My oversight.